backend-service/package.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "backend-service",
+  "version": "1.0.0",
+  "main": "index.js",
+  "scripts": {
+    "build": "npx tsc",
+    "start": "npx tsc && node dist/app.js",
+    "test": "vitest"
+  },
+  "keywords": [],
+  "author": "",
+  "license": "ISC",
```

Comment:
Add this line of config in: 

  ```json 
"type": "module", 
```

It's required to make `import` and `export` work with the tsconfig changes I'm suggesting.